front/app/components/FormBuilder/components/FormBuilderSettings/utils.tsx

```diff
@@ -66,6 +66,9 @@ const getBuiltInFieldStringKey = (
 ): MessageDescriptor | undefined => {
   let translatedStringKey: MessageDescriptor | undefined;
   switch (key) {
+    case 'title_multiloc':
```

Comment:
Noticed some missing copy/icons for title_multiloc field. Added the changes in this PR.

Reply:
Nice 🎉